input/pagecontent/sexgender.md

```diff
@@ -0,0 +1,336 @@
+    ...
+    {
+      "url" : "http://hl7.org/fhir/StructureDefinition/individual-recordedSexOrGender",
+      "extension" : [
+        {
+          "url" : "value",
+          "valueCodeableConcept" : {
+            "coding" : [
+              {
+                "system" : "http://hl7.org/fhir/administrative-gender",
+                "code" : "female"
+              }
+            ]
+          }
+        },
+        {
+          "url" : "type",
+          "valueCodeableConcept" : {
+            "coding" : [
+              {
+                "system" : "http://loinc.org",
+                "code" : "46098-0"
+              }
+            ]
+          }
+        },
+        {
+          "url" : "effectivePeriod",
+          "valuePeriod" : {
+            "start" : "2024-12-25"
+          }
+        },
+        {
+          "url" : "sourceDocument",
+          "valueCodeableConcept" : {
+            "coding" : [
+              {
+                "system" : "http://terminology.hl7.org.au/CodeSystem/rsg-source-document-type",
+                "code" : "passport"
+              }
+            ]
+          }
+        },
+        {
+          "url" : "sourceField",
+          "valueString" : "Sex"
+        },
+        {
+          "url" : "jurisdiction",
+          "valueCodeableConcept" : {
+            "coding" : [
+              {
+                "system" : "urn:iso:std:iso:3166",
+                "code" : "AU",
+                "display" : "Australia"
+              }
+            ]
+          }
+        }
+      ]
+    }
+  ],
+  ...
+}  
+~~~
+
+
+### Sex Assigned at Birth
+AU Base supports representation and exchange of the Sex Assigned at Birth data element (as defined in the [HL7 Cross Paradigm Implementation Guide: Gender Harmony - Sex and Gender Representation, Edition 1](https://hl7.org/xprod/ig/uv/gender-harmony/informative1/)) in:
+- [AU Base Patient](StructureDefinition-au-patient.html)
+- [AU Base Practitioner](StructureDefinition-au-practitioner.html)
+- [AU Base Related Person](StructureDefinition-au-relatedperson.html)
+
+Sex Assigned at Birth is represented with the [Person Recorded Sex or Gender](https://hl7.org/fhir/extensions/StructureDefinition-individual-recordedSexOrGender.html) extension:
+- the type element is populated with 1515311000168102\|Biological sex at birth\|
+- the value element is constrained to be [Biological Sex](https://healthterminologies.gov.au/fhir/ValueSet/biological-sex-1) ([extensible](http://hl7.org/fhir/R4/terminologies.html#extensible))
+
+When populating the value element of the [Person Recorded Sex or Gender](https://hl7.org/fhir/extensions/StructureDefinition-individual-recordedSexOrGender.html) extension for Sex Assigned at Birth:
+  - *Male* may be represented by sending the SNOMED CT code 248153007\|Male\|
+  - *Female* may be represented by sending the SNOMED CT code 248152002\|Female\|
+  - *Intersex* may be represented by sending the SNOMED CT code 33791000087105\|Intersex\|
```

Comment:
Incorrect code for Intersex - change to 32570691000036108

Reply:
fixed

Reply:
Confirmed fix